app/views/games/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<br>
+
+<div class="col-xs-4 col-xs-offset-1">
```

Comment:
with `bootstrap` you want to make sure your `col`s are *always* wrapped with `<div class="row"></div>`